boardgamegeek/api.py

```diff
@@ -888,17 +888,23 @@ def game(self, name=None, game_id=None, choose=BGGChoose.FIRST, versions=False,
         while added_items and len(game.comments) < total:
             page += 1
 
+            params['page'] = page
             xml_root = request_and_parse_xml(self.requests_session,
                                              self._thing_api_url,
-                                             params={"id": game_id,
-                                                     "pagesize": 100,
-                                                     "comments": 1,
-                                                     "page": page})
+                                             params=params,
+                                             timeout=self._timeout,
+                                             retries=self._retries,
+                                             retry_delay=self._retry_delay)
+
+            xml_root = xml_root.find("item")
+            if xml_root is None:
+                msg = "invalid data for game id: {}{}".format(game_id, "" if name is None else " ({})".format(name))
+                raise BGGApiError(msg)
 
-            added_items = add_game_comments_from_xml(game, xml_root)
+            added_items, total = add_game_comments_from_xml(game, xml_root)
```

Comment:
Changed to match the earlier call to `add_game_comments_from_xml` (otherwise `added_items` is a tuple like `(True, 493)`